sdk/Lusid.Sdk.Tests/tutorials/Ibor/GenericReconciliation.cs

```diff
@@ -0,0 +1,841 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Ibor
+{
+    [TestFixture]
+    public class GenericReconciliation : TutorialBase
+    {
+        private InstrumentLoader _instrumentLoader;
+        private IList<string> _instrumentIds;
+        private readonly string _portfolioOneScope = "testPortfolio1" + new Guid();
+        private readonly string _portfolioTwoScope = "testPortfolio2" + new Guid();
+        private readonly string _portfolioCode = Guid.NewGuid().ToString();
+        private DateTimeOffset _effectiveAt;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = _instrumentLoader.LoadInstruments();
+        }
+
+        /// <summary>
+        /// Perform a reconciliation on two identical portfolios except for the Address Key for trader name being scope dependent.
+        /// I.e., "Transaction/testPortfolio1/TraderName" = "John Doe" and "Transaction/testPortfolio2/TraderName" = "John Doe"
+        /// Unless a mapping rule is provided between these two address keys in the reconciliation then both keys will appear
+        /// as fail to match in the aggregation.
+        /// </summary>
+        [LusidFeature("F20-2")]
+        [Test]
+        public void RemappingProperties()
+        {
+            var quotePrice = 105m;
+            var units = "GBP";
+            var transactionDate = new DateTimeOffset(2022, 2, 1, 0, 0, 0, TimeSpan.Zero); // date of transaction
+            var traderName = "John Doe";
+
+            // Generate two identical portfolios
+            var valuationRequestOne = GeneratePortfolioTransactions(_portfolioOneScope, _portfolioCode, transactionDate,
```

Comment:
what is this returning? its not obvious to the reader, add some commentary or split out the steps

Reply:
will comment to make clearer that this is returning a valuation request for a single equity portfolio